source/base/timer.cc

```diff
@@ -56,8 +57,10 @@ Timer::Timer()
 #endif
 {
 #ifdef DEAL_II_WITH_MPI
-  mpi_data.sum = mpi_data.min = mpi_data.max = mpi_data.avg = numbers::signaling_nan<double>();
-  mpi_data.min_index = mpi_data.max_index = numbers::invalid_unsigned_int;
+  mpi_data.sum = mpi_data.min = mpi_data.max = mpi_data.avg = 0.;
```

Comment:
That's fine. I am just wondering if we had a reason to use invalid value to initialize these variable.